### PR TITLE
Add cluster ID as environment variable for containers

### PR DESCRIPTION
**Pull request by pheanex**
Add _PLATFORM_CLUSTER environment variable

So the json-file log-driver can add its content to log-lines

### CODE REVIEW

**Comment by aermakov-zalando** [comment]
Can we call it `_PLATFORM_CLUSTER_ID`? It might be more clear for the users.

**Comment by aermakov-zalando** [comment]
You might also want to update [the tests](https://github.com/zalando-incubator/kubernetes-on-aws/blob/f4c5bedef8b882d58bbcacc97a1f288f7569eed7/test/e2e/admission_controller.go#L94).

File: cluster/node-pools/master-default/userdata.yaml [hunk 1]
```diff
@@ -201,6 +201,7 @@ write_files:
 {{- if eq .Cluster.ConfigItems.teapot_admission_controller_inject_environment_variables "true" }}
             - --pod-enable-env-inject
             - --pod-inject-env=_PLATFORM_ACCOUNT={{ .Cluster.Alias }}
+            - --pod-inject-env=_PLATFORM_CLUSTER_ID={{ .Cluster.id }}
```
Comment (aermakov-zalando) [inline cluster/node-pools/master-default/userdata.yaml]: Should be `ID` ([source](https://github.com/zalando-incubator/cluster-lifecycle-manager/blob/master/api/cluster.go#L21)).

**Comment by pheanex** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1:

**Comment by hjacobs** [comment]
Ready for review?

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**